<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vanishing Point</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClearBackground"/>
    <addaction name="actionOpenImage"/>
    <addaction name="separator"/>
    <addaction name="actionClearLines"/>
    <addaction name="actionLoadLines"/>
    <addaction name="actionSaveLines"/>
    <addaction name="separator"/>
    <addaction name="actionClearSilhouette"/>
    <addaction name="actionLoadSilhouette"/>
    <addaction name="actionSaveSilhouette"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <widget class="QMenu" name="menuPen">
     <property name="title">
      <string>Pen</string>
     </property>
     <addaction name="actionPenVanishingLine"/>
     <addaction name="actionPenSilhouette"/>
    </widget>
    <widget class="QMenu" name="menuRendering">
     <property name="title">
      <string>Rendering</string>
     </property>
     <addaction name="actionRenderingLine"/>
     <addaction name="actionRenderingSilhouette"/>
     <addaction name="actionRenderingBasic"/>
     <addaction name="actionRenderingSSAO"/>
    </widget>
    <widget class="QMenu" name="menuGrammar">
     <property name="title">
      <string>Grammar</string>
     </property>
     <addaction name="actionGrammarMass"/>
     <addaction name="actionGrammarFacade"/>
    </widget>
    <addaction name="actionComputeCameraMatrix"/>
    <addaction name="separator"/>
    <addaction name="menuRendering"/>
    <addaction name="actionTextureMapping"/>
    <addaction name="separator"/>
    <addaction name="action3DReconstruction"/>
    <addaction name="action3DReconstructionAll"/>
    <addaction name="separator"/>
    <addaction name="menuPen"/>
    <addaction name="menuGrammar"/>
    <addaction name="separator"/>
    <addaction name="actionOption"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTool"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpenImage">
   <property name="text">
    <string>Open Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionSaveLines">
   <property name="text">
    <string>Save Lines</string>
   </property>
  </action>
  <action name="actionComputeVanishingPoint">
   <property name="text">
    <string>Compute Vanishing Point</string>
   </property>
  </action>
  <action name="actionLoadLines">
   <property name="text">
    <string>Load Lines</string>
   </property>
  </action>
  <action name="actionComputeCameraMatrix">
   <property name="text">
    <string>Compute Camera Matrix</string>
   </property>
  </action>
  <action name="actionClearLines">
   <property name="text">
    <string>Clear Lines</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCentering">
   <property name="text">
    <string>Centering</string>
   </property>
  </action>
  <action name="actionOption">
   <property name="text">
    <string>Option</string>
   </property>
  </action>
  <action name="action3DReconstruction">
   <property name="text">
    <string>3D Reconstruction</string>
   </property>
  </action>
  <action name="actionLoadSilhouette">
   <property name="text">
    <string>Load Silhouette</string>
   </property>
  </action>
  <action name="actionSaveSilhouette">
   <property name="text">
    <string>Save Silhouette</string>
   </property>
  </action>
  <action name="actionPenVanishingLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Vanishing Line</string>
   </property>
  </action>
  <action name="actionPenSilhouette">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Silhouette</string>
   </property>
  </action>
  <action name="action3DReconstructionAll">
   <property name="text">
    <string>3D Reconstruction for All</string>
   </property>
  </action>
  <action name="actionClearSilhouette">
   <property name="text">
    <string>Clear Silhouette</string>
   </property>
  </action>
  <action name="actionClearBackground">
   <property name="text">
    <string>Clear Background</string>
   </property>
  </action>
  <action name="actionTextureMapping">
   <property name="text">
    <string>Texture Mapping</string>
   </property>
  </action>
  <action name="actionRenderingLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionRenderingSilhouette">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Silhouette</string>
   </property>
  </action>
  <action name="actionRenderingBasic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Basic</string>
   </property>
  </action>
  <action name="actionGrammarMass">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Building Mass</string>
   </property>
  </action>
  <action name="actionGrammarFacade">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Facade</string>
   </property>
  </action>
  <action name="actionRenderingSSAO">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SSAO</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
